RecordCollection/Controllers/AlbumsController.cs

```diff
@@ -15,8 +19,39 @@ public AlbumsController(RecordCollectionContext context)
         public IActionResult Index()
         {
             var albums = _context.Albums.ToList();
```

Comment:
I would also add a null check in the Index action to check that the var albums is not null before passing it to the View. This will gracefully handle situations where the variable is null and prevent the application from breaking entirely.